packages/opentelemetry-instrumentation-langchain/opentelemetry/instrumentation/langchain/__init__.py

```diff
@@ -37,10 +37,24 @@ def __init__(
         exception_logger=None,
         disable_trace_context_propagation=False,
         use_legacy_attributes: bool = True,
+        metadata_key_prefix: Optional[str] = None
```

Comment:
this should default to the way it works tdoay

Reply:
It defaulted from Config, but I also added here for clarity